scripts/Imaging/imgpy/tests/test_helper.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import numpy.testing as npt
+
+backup_mp_avail = None
+g_shape = (10, 8, 10)
+
+
+def gimme_shape():
+    return g_shape
+
+
+def gen_img_numpy_rand():
+    return np.random.rand(*g_shape)
+
+
+def gen_img_shared_array_and_copy(shape=g_shape):
+    arr = gen_img_shared_array(shape)
+    copy = deepcopy(arr)
+    return arr, copy
+
+
+def gen_img_shared_array(shape=g_shape):
+    from parallel import utility as pu
+    d = pu.create_shared_array(shape)
+    n = np.random.rand(shape[0], shape[1], shape[2])
+    d[:] = n[:]
+
+    return d
+
+
+def gen_img_shared_array_with_val(val=1., shape=g_shape):
+    from parallel import utility as pu
+    d = pu.create_shared_array(shape)
+    n = np.full(shape, val)
+    d[:] = n[:]
+
+    return d
+
+
+def assert_equals(thing1, thing2):
+    npt.assert_equal(thing1, thing2)
+
+
+def assert_not_equals(thing1, thing2):
+    npt.assert_raises(AssertionError, npt.assert_equal, thing1, thing2)
+
+
+def deepcopy(source):
+    from copy import deepcopy
+    return deepcopy(source)
+
+
+def debug(switch=True):
+    if switch:
+        import pydevd
+        pydevd.settrace(
+            'localhost', port=59003, stdoutToServer=True, stderrToServer=True)
+
+
+def vsdebug():
+    import ptvsd
+    ptvsd.enable_attach("my_secret", address=('0.0.0.0', 59003))
+    print("Waiting for remote debugger at localhost:59003")
+    #Enable the below line of code only if you want the application to wait untill the debugger has attached to it
+    ptvsd.wait_for_attach()
```

Comment:
this function will only be added if there's an actual need for debugging, I have a flag on the function above `def debug(switch=True):` and it's never been used